locales/ml_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -67,23 +67,23 @@ msgstr "ഞങ്ങളെ %s"
 
 msgctxt "SERP footer content"
 msgid "%s%s%s Billion Searches"
-msgstr ""
+msgstr "%s%s%s ലക്ഷം കോടി തിരച്ചിലുകൾ"
 
 msgctxt "noresults"
 msgid "%sClick here%s to try again, if you think there should be results for this search."
 msgstr ""
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo!%s"
-msgstr ""
+msgstr "%sDuckDuckGo! പ്രചരിപ്പിക്കാൻ സഹായിക്കൂ%s"
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo%s."
-msgstr ""
+msgstr "%sDuckDuckGo പ്രചരിപ്പിക്കാൻ സഹായിക്കൂ%s"
 
 msgctxt "frontpage"
 msgid "%sLearn More%s."
-msgstr ""
+msgstr "%കൂടൂതൽ അറിയാൻ%s."
```

Comment:
Hi @athimannil! I think this is good to go, once we fix this placeholder:

```suggestion
msgstr "%sകൂടൂതൽ അറിയാൻ%s."
```

Reply:
@jbarrett fixed :-) Thanks for your review 👍